stdlib/public/core/SIMDVectorTypes.swift.gyb

```diff
@@ -109,6 +109,14 @@ public extension SIMD${n} where Scalar : FixedWidthInteger {
   }
 }
 
+extension SIMD${n} : CustomDebugStringConvertible {
+  /// Debug string representation
```

Comment:
This overrides the default documentation for debugDescription. Was that intentional?

Reply:
I pulled the fix into https://github.com/apple/swift/pull/21379